server/views/macros/prisonerSearchOffenceCodeDecisionInput.njk

```diff
@@ -0,0 +1,46 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% macro prisonerSearchOffenceCodeDecisionInput(selectedDecisionId, questionId, userId, prisonerName, userSearchInput, errors) %}
+    {% if selectedDecisionId == questionId and userId != null %}
+        <input type="hidden" name="userId{{ questionId }}" value="{{ userId }}" />
+        <div>
+            <p><span class="govuk-!-font-weight-bold">Name</span> - {{ prisonerName }} </p>
+            <p><span class="govuk-!-font-weight-bold">PRN</span> - {{ userId }} </p>
+            <div>
+                {{ govukButton({
+                    text: 'Delete',
+                    classes: "govuk-button--secondary",
+                    type: "submit",
+                    name: "deleteUser",
+                    value: "deleteUser"
+                }) }}
+            </div>
+        </div>
+    {% else %}
+        <div class="flex-container flex-center">
+            <div class="govuk-!-margin-right-3">
+                {{ govukInput({
+                    id: "userSearchInput" + questionId,
```

Comment:
We need the question id too added to all of the input ids, so that we can know if they are related to the currently selected decision, so we can only validate those that are attached to the currently selected decision.